charts/openfga/templates/deployment.yaml

```diff
@@ -30,10 +30,7 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
-      {{- with .Values.imagePullSecrets }}
-      imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
+      {{- include "openfga.pullSecrets" . | nindent 6 }}
```

Comment:
Thanks @borysei!

This feels like a breaking change that can affect many folks.

We can maybe support the previous value alongside the new values, and mark it as deprecated.

If that is not possible we need to document this well, and check in with the community to see how much appetite for a breaking change they may have